[2025-01-31T11:32:29.471+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.greet_task manual__2025-01-31T11:32:25.029341+00:00 [queued]>
[2025-01-31T11:32:29.485+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.greet_task manual__2025-01-31T11:32:25.029341+00:00 [queued]>
[2025-01-31T11:32:29.485+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-01-31T11:32:29.504+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): greet_task> on 2025-01-31 11:32:25.029341+00:00
[2025-01-31T11:32:29.512+0000] {standard_task_runner.py:57} INFO - Started process 94 to run task
[2025-01-31T11:32:29.517+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'greet_task', 'manual__2025-01-31T11:32:25.029341+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dags/first_dag.py', '--cfg-path', '/tmp/tmpmgkqaifj']
[2025-01-31T11:32:29.522+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask greet_task
[2025-01-31T11:32:29.700+0000] {task_command.py:416} INFO - Running <TaskInstance: first_dag.greet_task manual__2025-01-31T11:32:25.029341+00:00 [running]> on host 43b24753d50b
[2025-01-31T11:32:29.819+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='greet_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T11:32:25.029341+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T11:32:25.029341+00:00'
[2025-01-31T11:32:29.822+0000] {logging_mixin.py:154} INFO - Hello, Apache Airflow!
[2025-01-31T11:32:29.822+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-01-31T11:32:29.835+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=greet_task, execution_date=20250131T113225, start_date=20250131T113229, end_date=20250131T113229
[2025-01-31T11:32:29.891+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T11:32:29.915+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
